README.md

```diff
@@ -22,6 +21,7 @@ module "MLOps" {
   algorithm_choice        = "classification"
   sagemaker_training_notebook_instance_type = "ml.m4.xlarge"
   inference_instance_count = 1
+  preprocessing_script_path = "terraform\preprocess_data.py"
```

Comment:
Is this supposed to have a forward slash instead of backslash?

Reply:
fixed